common/terminology.md

```diff
@@ -0,0 +1,11 @@
+#h2 Pattern 
+<p>A high level solution to a common design problem. May use multiple components put together in different ways. Generally a basis for solving a design problem, not a prescriotive solution</p>
+<p>Examples: tabs, video controls, checkout steps 
+
+#h2 Component 
+<p>Specific chunks of the UI that can be put together to compose a page. Each component is prescriptive and has a specific role</p>
+<p>Examples: card, footer, banner, header
+
+#h2 Package
+<p>A block of code representing a component or some global code that can be installed in a product application through NPM</p>
+<p>Examples: article view, Nature CSS context, Global CSS utilities, card
```

Comment:
Rob, your lack of closing tags is killing me. 😂

Reply:
Just testing your spotting skillz 😆